src/main/java/net/sf/jabref/importer/fileformat/BibtexParser.java

```diff
@@ -283,7 +262,7 @@ public ParserResult parse() throws IOException {
                                     + be.getAuthorTitleYear(40) + " ("
                                     + Localization.lang("grouping may not work for this entry") + ")");
                         }
-                    } catch (IOException ex) {
+                    } catch (Exception ex) {
```

Comment:
If a bibtex field has for example no closing bracket, then another exception is thrown (not sure which concrete one, something along the lines of "I got something, but expected something else"). So I changed `IOException` to a general exception to handle also these cases. Without this change the test `parseIgnoresAndWarnsAboutCorruptedEntry` fails.